kubernetes/opencost.yaml

```diff
@@ -139,7 +166,27 @@ spec:
     spec:
       restartPolicy: Always
       serviceAccountName: opencost
+      volumes:
+        - name: nginx-conf
+          configMap:
+            name: nginx-conf
+            items:
+              - key: nginx.conf
+                path: default
       containers:
+        - image: cesign/opencost-ui:v0.12
```

Comment:
Do you have the dockerfile that goes with this image? Maybe add to the /ui folder?

Reply:
fixed